recipe/meta.yaml

```diff
@@ -76,34 +83,35 @@ test:
   source_files:
     - tests
   requires:
+    - pip
     - pytest
-  # TODO: add these tests when we start to include builds with turbo
-  # files:
-  #     - 687px-Mona_Lisa,_by_Leonardo_da_Vinci,_from_C2RMF_retouched.jpg
-  #     - ensure_jpegturbo_imagecodecs_plays_nicely_with_jpeg9.py
-  # requires:
-  #   - pillow
-  #   - opencv
   imports:
     - imagecodecs
     - imagecodecs._imagecodecs
   commands:
+    - pip check
     - cd tests
       # TODO: determine why OpenMP on Windows build workers is unreliable
-    - set SKIP_OMP=true                 [win]
+    - set SKIP_OMP=true                # [win]
     # https://github.com/conda-forge/imagecodecs-feedstock/pull/32#issuecomment-863041029
-    - pytest -vv test_imagecodecs.py -k "not test_image_roundtrips[jpeg2k-uint16-rgb-out-bytearray-None]"  #[not s390x]
-    - pytest -vv test_imagecodecs.py -k "not test_image_roundtrips[jpeg2k-uint16-rgb-out-bytearray-None] and not test_packints_decode and not test_floatpred and not test_float24 and not test_jpegsof3 and not test_image_roundtrips"  #[s390x]
+
+    # disable test_jpeg2k_numthreads on all platform except win: it causes segmentation fault.
+    
+    # disable completely tests on osx-64, as they fail at 18% with no message
+    # - pytest -vv test_imagecodecs.py -k "not (test_jpeg2k_numthreads or test_image_roundtrips[jpeg2k-uint16-rgb-out-bytearray-None])"  # [osx and (x86_64 or x86)]
```

Comment:
do we want to disable all tests, or would be better to find out which one fails and disable only that one ?

Reply:
I could investigate further, thanks @M-Waszkiewicz-Anaconda

Reply:
`[osx and x86_64x]` is enough instead of `[osx and (x86_64 or x86)]` as we already do not support 32-bit architecture https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#preprocessing-selectors